ipt_connect/IPTdev/models.py

```diff
@@ -794,5 +797,5 @@ def update_all(sender, **kwargs):
 	for pb in Problem.objects.all():
 		pb.update_scores()
 
-	return "Teams, participants and problems updated !"
+	return "Teams, participants and problems updated in " , int(time.time()-old_time) ,  " seconds !"
```

Comment:
So if I understand correctly, the goal is to trigger the update every X seconds, minutes, correct? (which will appear in another commit).

To think about: do we want the scores to update in the middle of a round, with teams having played a different numbers of PF with different roles, etc... and possibly the chairmen not entering the grades at the same time? (some wait until the end of a round so they can concentrate on the fights, other do it directly). That might result in a messy leaderboard.

Reply:
> So if I understand correctly, the goal is to trigger the update every X seconds, minutes, correct?

No. The goal is just to print how many time the update consumed. It is stated in the detailed commit message: "Just to scarify the administrator :)"

Reply:
Aaaah ok. I thought you were implemeting some kind of timer, but it makes more sense - I read too fast